test/protocol-buffer-conversion-test.ts

```diff
@@ -29,6 +33,18 @@ testPathElement.setCurrency(testCurrencyProto)
 testPathElement.setAccount(testAccountAddress)
 testPathElement.setIssuer(testAccountAddressIssuer)
 
+// test IssuedCurrencyAmount
+const testIssuedCurrency = new IssuedCurrencyAmount()
+testIssuedCurrency.setCurrency(testCurrencyProto)
+testIssuedCurrency.setIssuer(testAccountAddress)
+testIssuedCurrency.setValue('100')
+
+// test Invalid IssuedCurrencyAmount
```

Comment:
refactor line 154 to use this new object rather than creating it twice.

Reply:
nice catch